samples/csharp_dotnetcore/49.qnamaker-all-features/QnABotAllFeatures.csproj

```diff
@@ -8,9 +8,9 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.1" />
-    <PackageReference Include="Microsoft.Bot.Builder.AI.QnA" Version="4.9.2" />
-    <PackageReference Include="Microsoft.Bot.Builder.Dialogs" Version="4.9.2" />
-    <PackageReference Include="Microsoft.Bot.Builder.Integration.AspNet.Core" Version="4.9.2" />
+    <PackageReference Include="Microsoft.Bot.Builder.AI.QnA" Version="4.10.0-preview-200621-139841" />
+    <PackageReference Include="Microsoft.Bot.Builder.Dialogs" Version="4.10.0-preview-200621-139841" />
+    <PackageReference Include="Microsoft.Bot.Builder.Integration.AspNet.Core" Version="4.10.0-preview-200621-139841" />
```

Comment:
Production samples can't depend on packages that are not on Nuget. These packages are from our daily feed, which won't work here. This effectivly blocks this sample from merging.

Reply:
@vipeketi 

this PR no longer required with new preview feature approach.

please pull it back.